roles/pulp_content/templates/pulpcore-content.service.j2

```diff
@@ -14,10 +14,14 @@ User={{ pulp_user }}
 Group={{ pulp_group }}
 WorkingDirectory=/var/run/pulpcore-content/
 RuntimeDirectory=pulpcore-content
+
+# timeout 90 is needed Pulp to service its 1st request on extremely slow
+# machines, such as qemu-emulated 2-core machines
```

Comment:
I think it needs a new changelog entry

Reply:
I'll combine it with pulpcore-api, yeah. It may not even be needed for pulpcore-content, but I did it at the same time.

Reply:
@gerrod3 same here